routes/event.js

```diff
@@ -315,6 +358,77 @@ function sendUpdateEmail(event) {
 	});
 }
 
+function addGuestDetails(event, details, callback) {
+	var guestDetails = details;
+	var numInvitees = event.invitees.length;
+	var inviteeIndex = 0;
+
+	for (var i = 0; i < numInvitees; i++)
+	{	
+		var invitee = event.invitees[i];
+			
+		userMapper.findUserByEmail(invitee.email, function(err, user) {
+			if (invitee.state == 'Attending' && invitee.accessRequirements != null && invitee.dietaryRestrictions != null)
```

Comment:
This condition only sends an email if both requirements are present. So a guest with just dietary restrictions will be ignored. So if dietary OR access requirements are present then they should be added